client-app/pages/product.vue

```diff
@@ -173,14 +128,12 @@ watchEffect(() => {
     });
   }
 });
-</script>
 
-<style scoped lang="scss">
-@media print {
-  .product {
-    @apply min-w-[1024px] w-full;
-
-    zoom: 0.7;
-  }
-}
-</style>
+watch(
+  _relatedProducts,
+  (newVal) => {
+    relatedProducts.value = [...newVal];
+  },
+  { deep: true },
+);
+</script>
```

Comment:
Why we need this? It's exactly the same as just use `relatedProducts` directly

Reply:
That's because we use `shallowRef`, so changes will trigger update only if array was changed, not its elements. Need to find why do we need this, what changes product properties

Reply:
Let's fix bug first

Reply:
Not `shallowRef `. Fixed.